core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -614,10 +641,10 @@ OakRBuffer get(K key) {
 
         Chunk<K, V> c = findChunk(key); // find chunk matching key
         Chunk.LookUp lookUp = c.lookUp(key);
-        if (lookUp == null || lookUp.handle == null) {
+        if (lookUp == null || lookUp.valueSlice == null) {
```

Comment:
Sometimes you compare lookUp.valueReference with DELETED_VALUE and sometimes lookUp.valueSlice with null... What is the rule? What means what? Must be consistent...

Reply:
I looked through the code, and I never use the value reference in a condition.
Can you refer me to where you saw the use of value reference in a condition?